src/main/java/emu/grasscutter/command/commands/PermissionCommand.java

```diff
@@ -10,7 +10,7 @@
 
 import static emu.grasscutter.utils.Language.translate;
 
-@Command(label = "permission", usage = "permission <add|remove> <permission>", permission = "permission", description = "commands.permission.description")
+@Command(label = "permission", usage = "permission <add|remove> <permission>", permission = "server.permission", description = "commands.permission.description", targetRequirement = Command.TargetRequirement.NONE)
```

Comment:
This is a very dangerous change as server operators tend to hand out `server.*` permissions. Negative permissions make this an easy config fix but this would need to be **very** visible on the changelog.